doc/nrf/known_issues.rst

```diff
@@ -845,18 +851,53 @@ KRKNWK-9214: Pigweed submodule may not be accessible from some regions
 HomeKit
 =======
 
-.. rst-class:: v1-9-1 v1-9-0
+.. rst-class:: v2-0-0
 
-KRKNWK-13063: RTT logs do not work with the Light Bulb multiprotocol sample with DFU on nRF52840
-  The Light Bulb multiprotocol sample with Nordic DFU activated in debug version for nRF52840 platform does not display RTT logs properly.
+KRKNWK-14130: Bluetooth LE TX configuration is set to ``0`` dBm by default
+  The minimum Bluetooth LE TX configuration required is at least ``4`` dBm.
+  For HomeKit multiprotocol samples, this should be ``8`` dBm.
+  This results in a weak signal on the SoC itself.
 
-  **Workaround:** Disable RTT logs for the bootloader.
+  **Workaround:** You need to configure the appropriate dBm values for Bluetooth LE and Thread manually in the source code.
 
-.. rst-class:: v1-9-1 v1-9-0
+.. rst-class:: v2-0-0 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+
+KRKNWK-14081: HomeKit SDK light bulb example does not not work with MTD
+  If the MTD is set to ``y`` in the light bulb sample, user is not able to communicate with the device after it has been added to the Home app using an iPhone and a HomePod Mini.
+
+.. rst-class:: v2-0-0 v1-9-1 v1-9-0
 
 KRKNWK-13947: Net core downgrade prevention does not work on nRF5340
   HAP certification requirements are not met because of this issue.
 
+.. rst-class:: v2-0-0
+
+KRKNWK-13607: Stateless switch application crashes upon factory reset
+  When running Thread test suit on the stateless switch application, the CI crashes upon factory reset.
+
+.. rst-class:: v2-0-0 v1-9-1 v1-9-0
+
+KRKNWK-13249: Unexpected assertion in HAP Bluetooth Peripheral Manager
+  When Bluetooth LE layer emits callback with a connect or disconnect event, one of its parameters is an underlying Bluetooth LE connection object.
+  On rare occasions, this connection object is no longer valid by the time it is processed in HomeKit, and this results in assertion.
+  There is no proven workaround yet.
+
+.. rst-class:: v2-0-0 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+
+KRKNWK-11729 Stateless switch event characteristic value not handled according to spec in Bluetooth LE mode
+  The stateless programmable switch application does not handle the value of the stateless switch event characteristic in the Bluetooth LE mode according to the specification.
+  According to the specification, the accessory is expected to return null once the characteristic has been read or after 10 seconds have passed.
+  In the current implementation in the |NCS|, the characteristic value does not change to null right after it was read, and changes to null after 5 seconds instead.
```

Comment:
```suggestion
  In the current |NCS| implementation, the characteristic value does not change to null immediately after it is read, and changes to null after 5 seconds instead.
```

Reply:
Kept the first part unchanged, rest applied.